### ISSUE TITLE
Wikipedia Molecules' URL needs updating

### ISSUE THREAD

**Issue by mjw99** [thsa/datawarrior#8]
The Wikipedia Molecules database has stopped working. This is because the URL http://www.cheminfo.org/wikipedia/idcode.txt is returning a 301

The following line:

https://github.com/thsa/datawarrior/blob/11717dcabbc6f6792530d0bbcbc59731f9129c8f/src/com/actelion/research/datawarrior/task/db/DETaskRetrieveWikipediaCompounds.java#L26

needs changing to 
```java
 private static final String RAW_DATA_URL = "https://wikipedia.cheminfo.org/idcode.txt";
```

**Comment by thsa** [comment]
many thanks. Is now updated...
________________________________
Von: Mark J. Williamson ***@***.***>
Gesendet: Montag, 8. August 2022 15:28
An: thsa/datawarrior ***@***.***>
Cc: Subscribed ***@***.***>
Betreff: [thsa/datawarrior] Wikipedia Molecules' URL needs updating (Issue #8)


The Wikipedia Molecules database has stopped working. This is because the URL http://www.cheminfo.org/wikipedia/idcode.txt<https://che01.safelinks.protection.outlook.com/?url=http%3A%2F%2Fwww.cheminfo.org%2Fwikipedia%2Fidcode.txt&data=05%7C01%7Cthomas.sander%40idorsia.com%7C376e9c48ef7c479fdb6508da7941e652%7Cbb9214bf0cb941fdbd55d0c1c3eda110%7C0%7C0%7C637955621183337280%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=5fswxa0z5NHMGKKCxGw04UJjiFmIXOHnjoYuU1LTR4s%3D&reserved=0> is returning a 301

The following line:

https://github.com/thsa/datawarrior/blob/11717dcabbc6f6792530d0bbcbc59731f9129c8f/src/com/actelion/research/datawarrior/task/db/DETaskRetrieveWikipediaCompounds.java#L26<https://che01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fthsa%2Fdatawarrior%2Fblob%2F11717dcabbc6f6792530d0bbcbc59731f9129c8f%2Fsrc%2Fcom%2Factelion%2Fresearch%2Fdatawarrior%2Ftask%2Fdb%2FDETaskRetrieveWikipediaCompounds.java%23L26&data=05%7C01%7Cthomas.sander%40idorsia.com%7C376e9c48ef7c479fdb6508da7941e652%7Cbb9214bf0cb941fdbd55d0c1c3eda110%7C0%7C0%7C637955621183337280%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=LYk2Qww4YiSmBm%2F2SXJzTGwZJeazobIYd7q5f5jVGjg%3D&reserved=0>

needs changing to

 private static final String RAW_DATA_URL = "https://wikipedia.cheminfo.org/idcode.txt";

—
Reply to this email directly, view it on GitHub<https://che01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fthsa%2Fdatawarrior%2Fissues%2F8&data=05%7C01%7Cthomas.sander%40idorsia.com%7C376e9c48ef7c479fdb6508da7941e652%7Cbb9214bf0cb941fdbd55d0c1c3eda110%7C0%7C0%7C637955621183337280%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=IDM3kNTGScsLxOUx1ZXAcZ1VWhe7UxWsPyGDTW%2BZ0mo%3D&reserved=0>, or unsubscribe<https://che01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FACNFEBQ2AQ2VFRZHHCLWPNDVYEDQHANCNFSM555EKZ7A&data=05%7C01%7Cthomas.sander%40idorsia.com%7C376e9c48ef7c479fdb6508da7941e652%7Cbb9214bf0cb941fdbd55d0c1c3eda110%7C0%7C0%7C637955621183337280%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=E6jFA28qh8jIT8FBDxOq6hw5wgojefT7r%2BRJO4ZTc7c%3D&reserved=0>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

________________________________
The information of this email and in any file transmitted with it is strictly confidential and may be legally privileged. It is intended solely for the addressee. If you are not the intended recipient, any copying, distribution or any other use of this email is prohibited and may be unlawful. In such case, you should please notify the sender immediately and destroy this email. The content of this email is not legally binding unless confirmed by letter. Any views expressed in this message are those of the individual sender, except where the message states otherwise and the sender is authorized to state them to be the views of the sender's company.

**Comment by mjw99** [comment]
Fixed in ed5b7fca
